arm-iothub/2016-02-03/swagger/iothub.json

```diff
@@ -1794,7 +1774,7 @@
     "api-version": {
       "name": "api-version",
       "in": "query",
-      "description": "Version of the Api.",
+      "description": "The version of the API.",
       "required": true,
       "type": "string",
       "default": "2016-02-03"
```

Comment:
please remove the default. It is not necessary. Clients will take this value any ways.